discord/ext/commands/help.py

```diff
@@ -272,7 +272,7 @@ class HelpCommand:
     show_hidden: :class:`bool`
         Specifies if hidden commands should be shown in the output.
         Defaults to ``False``.
-    verify_checks: :class:`bool`
+    verify_checks: Union[:class:`bool`, :class:`None`]
```

Comment:
```suggestion
    verify_checks: Optional[:class:`bool`]
```

The ``None`` behaviour is completely undocumented.

Reply:
No worries, was uhming and ahing about how to document a tribool